src/DataGlobals.hpp

```diff
@@ -40,8 +41,10 @@ class RaspStore;
  * replaced in another program.
  */
 namespace DataGlobals {
-std::shared_ptr<RaspStore> GetRasp();
-void SetRasp(std::shared_ptr<RaspStore> rasp);
+  std::shared_ptr<RaspStore> GetRasp();
```

Comment:
This changes the indent

Reply:
Yes, the indent is changed. However, contents of namespace DataGlobals  should be indented. See whole namespace DataGlobals. If you don't think so, I am glad to make it un-indented.

Reply:
Disregard. Change as little as possible.